robottelo/hosts.py

```diff
@@ -14,6 +15,12 @@
 logger = logging.getLogger('robottelo')
 
 
+def setup_capsule(satellite, capsule):
```

Comment:
Why create this method that has to be imported and doesn't return the result, instead of directly calling `satellite.capsule_certs_generate` where needed?

Reply:
This was a todo that shouldn't have made it into the pr.